### ISSUE TITLE
"(Authenticated)" in source list is misleading

### ISSUE THREAD

**Issue by DarwinJS** [chocolatey/choco#946]
I have noticed that other IT folks using chocolatey are confused by this label - in one of two ways:
*) That "Authenticated" means an authentication check was performed when adding the source.
*) That "Authenticated" possibly means an ongoing state of authentication to the feed or that it is periodically checked (say when listed).

I think it would be more clear to have a label simply indicating that auth info is registered with the source.  Something similar to "(w/ Auth Info)" - but properly word smithed.

**Comment by ferventcoder** [comment]
Apologies, it comes from the terminology "Authenticated source" that has been used with NuGet/Chocolatey for years. I can see how it may be confusing for folks not familiar with the terminology. Let's change it to use something better. 

Note for me - `choco source -r` should still output authenticated so it doesn't break any machine parsing.

tl;dr - Naming is hard.

**Comment by DarwinJS** [comment]
I agree naming is hard - it has to be concise, express the correct meaning and nothing more.  I find I can get it "more correct" when I give hard thought to expressing only the exact meaning of what I am trying to say with no unintended implications (semantics).  

Back to the old "If I had more time I would have written you a shorter letter."

**Comment by ferventcoder** [comment]
TBH I didn't give any thought to it as that is what NuGet has called them. But now I will 👍
